src/allDrugs.jl

```diff
@@ -14,7 +14,7 @@ function getODEparamsAll(p::Array{Float64, 1}, concentrations::Array{Float64, 2}
         effects[4, :, i] = p[k + 5] .* xx
         k += 6
     end
-    effects[5, :, :] .= p[27]
+    effects[5, :, :] .= p[27] #percentage in G1
     effects[6, :, :] .= floor(p[28]) #nG1
```

Comment:
It'd be clearer to use the labelled names here, rather than indices, since you have them.

Reply:
Sure. Will do.